src/components/semantic/presenters/FigurePresenter.tsx

```diff
@@ -99,12 +98,12 @@ export function FigurePresenter(props: PresenterProps<Figure>) {
         reader.onload = async function() {
             const src = await githubUpload(appContext, basePath, file.name, reader.result as string);
             if (src) {
-                setReplacedFile(src === docRef.current.src);
-                setLastUpdated(Date.now());
-                update({
-                    ...docRef.current,
-                    src,
-                });
+                if (src === docRef.current.src) {
+                    setReplacedFile(true);
+                    setTimeout(() => { setReplacedFile(false) }, 3000);
```

Comment:
This is an addition to make the replaced image banner disappear after 3s. It is possible that that might cause confusion

Reply:
I think the banner should stay IMO. If content say that it's in the way at some point, we could put a cross in the top corner, allowing them to dismiss it themselves.